common/libs/inputValidator.js

```diff
@@ -0,0 +1,15 @@
+export function isInt(n) {
```

Comment:
doesnt seem like this file is used? I would love better name also - this code has nothing to do with inputs